Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -131,11 +131,53 @@ private string ConvertJToken(JToken jToken)
                         string => $"\"{value}\"",
                         _ => value?.ToString()?.Replace(',', '.'), // Replace decimal commas.
                     };
+
                 case JArray jArray:
-                    return $"new[] {{ {string.Join(", ", jArray.Select(ConvertJToken))} }}";
+                    var format = $"{string.Join(string.Empty, Enumerable.Repeat(" ", indentationDepth + 4))}";

```

Comment:
```suggestion
                    var indentation = $"{string.Join(string.Empty, Enumerable.Repeat(" ", indentationDepth + 4))}";
```

Reply:
You can simplify this by using [this `string` constructor](https://docs.microsoft.com/en-us/dotnet/api/system.string.-ctor?view=net-6.0#system-string-ctor(system-char-system-int32)):
```suggestion
                    var indentation = new string(' ', indentationDepth + 4)";
```
The same goes for the other occurence.

Reply:
@devlife Did you see these suggestions?

Reply:
Ah no I did not. I replaced the string.Join statements with the string ctor.